openmmtools/multistate/sams.py

```diff
@@ -171,10 +172,14 @@ def __init__(self,
                  log_target_probabilities=None,
                  state_update_scheme='global-jump',
                  locality=5,
-                 update_stages='two-stage',
-                 flatness_criteria='logZ-flatness',
-                 flatness_threshold=0.2,
+                 weight_update=True,
                  weight_update_method='rao-blackwellized',
+                 update_stages='two-stage',
+                 flatness = None,
```

Comment:
`flatness` refers to the histogram so I replaced it by `histogram_flatness`. I also replaced `logZ_flatness` by `minimum_logZ` because the corresponding criteria is satisfied if a minimum logZ value is attained and this is not related to the flatness in logZ.